boms/integration-tests/src/test/java/com/google/cloud/MaximumLinkageErrorsTest.java

```diff
@@ -49,34 +42,15 @@ public void textMaximumLinkageErrors()
 
     Path bomFile = Paths.get("../cloud-oss-bom/pom.xml");
     Bom bom = RepositoryUtility.readBom(bomFile);
-    
-    ImmutableSetMultimap<SymbolProblem, ClassFile> oldProblems = findProblems(baseline);
-    ImmutableSetMultimap<SymbolProblem, ClassFile> newProblems = findProblems(bom);
+
+    ImmutableSetMultimap<SymbolProblem, ClassFile> oldProblems =
+        LinkageChecker.create(baseline).findSymbolProblems();
```

Comment:
Where do we call create and not immediately call findSymbolProblems? or reuise a linkage checker?

Reply:
LinkageCheckerRule (the enforcer rule):
```
        LinkageChecker linkageChecker = LinkageChecker.create(classpath, directDependencies);
        ImmutableSetMultimap<SymbolProblem, ClassFile> symbolProblems =
            linkageChecker.findSymbolProblems();
        if (reportOnlyReachable) {
          ClassReferenceGraph classReferenceGraph = linkageChecker.getClassReferenceGraph();

```